src/assets/scss/components/_header.scss

```diff
@@ -1,3 +1,7 @@
+
```

Comment:
@vihan85 
Check format code. Xuống dòng quá nhiều

Reply:
Hi @huulochuynh
This issue has been resolved

Thanks best & regards